lld/test/COFF/nodefaultlib.test

```diff
@@ -1,5 +1,6 @@
-# RUN: cp %p/Inputs/hello64.obj %T
-# RUN: cp %p/Inputs/std64.lib %T
+# RUN: mkdir -p %t.dir
```

Comment:
Could one just do `mkdir -p %t` here?   Or is there something else with that name already?

Reply:
I believe we could just `mkdir -p %t`. Using a `%t.<suffix>` is the typical convention though from what I've seen and matches everything else in the test.